libs/execution_context/ebpf_maps.c

```diff
@@ -19,6 +19,18 @@ typedef struct _ebpf_core_map
     uint8_t* data;
 } ebpf_core_map_t;
 
+// Issue - This code path should include ebpf_api.h for ebpf_map_option_t, but
+// can't due to name conflicts.
```

Comment:
Move this to ebpf_structs.h next to `ebpf_map_type_t` which is already used by both KM and UM.